server/sonar-web/src/main/js/app/components/nav/app.js

```diff
@@ -24,13 +24,13 @@ import ReactDOM from 'react-dom';
 import GlobalNav from './global/global-nav';
 import ComponentNav from './component/component-nav';
 import SettingsNav from './settings/settings-nav';
-import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../api/nav';
+import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../../api/nav';
 
 export default class App {
   start () {
     const options = window.sonarqube;
 
-    require('../../components/workspace/main');
+    require('../../../components/workspace/main');
 
     return new Promise((resolve) => {
```

Comment:
Thanks just wanted to make sure I wasn't doing anything wrong with https://github.com/trajano/trajano/commit/6d43283d225cb97428922975265442c476de22b6